x-pack/plugins/synthetics/public/legacy_uptime/components/monitor_management/action_bar/action_bar.tsx

```diff
@@ -139,11 +146,23 @@ export const ActionBar = ({
             <EuiFlexItem grow={false}>
               <WarningText>{!isValid && hasBeenSubmitted && VALIDATION_ERROR_LABEL}</WarningText>
             </EuiFlexItem>
+
+            <EuiToolTip
+              position="top"
+              content={<p>Works on any kind of element &mdash; buttons, inputs, you name it!</p>}
+            >
+              <EuiButton disabled={true} onClick={() => {}}>
+                Hover me
+              </EuiButton>
+            </EuiToolTip>
+
             {onTestNow && (
               <EuiFlexItem grow={false}>
                 {/* Popover is used instead of EuiTooltip until the resolution of https://github.com/elastic/eui/issues/5604 */}
                 <EuiPopover
                   repositionOnScroll={true}
+                  ownFocus={false}
+                  initialFocus={''}
```

Comment:
What does this do when a value isn't specified?

Reply:
Here we don't want the popover to capture focus when it's opened, to behave like a tooltip. Providing an empty _selector_ is the only option as `undefined` is the default value.